include/drivers/st25r3911b_nfc.rst

```diff
@@ -0,0 +1,84 @@
+.. _st25r3911b_nfc_readme:
+
+NFC Reader ST25R3911B
+#####################
+
+NFC Reader ST25R3911B can be used to read/write in ISO/IEC 14443 A compatible NFC tags,
+creating an NFC polling device. The library is intended to use with the NFC reader ST25R3911B
+Nucleo expansion board.
+
+NFC Reader ST25R3911B
+*********************
+
+The ST25R3911B is an Arduino shield which can be used to communicate over NFC.It can generate its own
+field to read and write NFC tags and behave like a tag itself. The ST25R3911B is a slave devices and
+the external microcontroller initiaties all communication. Communication is performed by 4-wire
+Serial Pheripheral Interface (SPI). The ST25R3911B sends an interrupt request (pin IRQ) to the microcontroller. For the documentation of the NFC chip, see `ST25R3911B chip documentation <https://www.st.com/resource/en/datasheet/st25r3911b.pdf>`_
+
+Library functionality
+*********************
+
+The library provides a following functionality:
+
+* Detecting ISO/IEC 14443 A tags and reading their UID
+* Performing Collision Avoidance Procedure
```

Comment:
In nRF52 documentation the term _Automatic Collision Resolution_ is used. Maybe this can be clarified with tech writers, but I think it would be good to use consistent terminology to avoid confusing the customers.
This comment potentially applies to other places in this documentation.

Reply:
changed